downstream/assemblies/hub/assembly-validated-content.adoc

```diff
@@ -0,0 +1,75 @@
+[id="assembly-validated-content"]
+= {Valid}
+
+{PlatformName} includes {Valid}, which complements existing {CertifiedCon}. 
+
+{Valid} provides an expert-led path for performing operational tasks on a variety of platforms including both Red Hat and our trusted partners.
+
+== Configuring validated or certified collections with the installer
+
+When you download and run the bundle installer, certified and validated collections are automatically uploaded. 
+Certified colections are uploaded into the `rh-certified` repository, while validated collections are uploaded into the `validated` repository. 
```

Comment:
```suggestion
Certified collections are uploaded into the `rh-certified` repository, while validated collections are uploaded into the `validated` repository. 
```

For the whole sentence: Certified collections are uploaded into the rh-certified repository. Validated collections are uploaded into the validated repository.